dts/bindings/pinctrl/nxp,mcux-rt-pinctrl.yaml

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) 2021 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The node has the 'pinctrl' node label set in MCUX RT SoC's devicetree. These
+  node can be autogenerated using the MCUXpresso config tools combined with
+  the parse-rt-mex.py script in the NXP HAL.
+
+
+compatible: "nxp,mcux-rt-pinctrl"
+
+include:
+    - name: base.yaml
+    - name: pincfg-node-group.yaml
+      child-binding:
+        property-allowlist:
+          - drive-open-drain
+
+properties:
+    reg:
+      required: true
+
+child-binding:
+  description: |
+    MCUX RT pin controller pin configuration group.
+  child-binding:
+    properties:
+      pinmux:
+        required: true
+        type: array
+        description: |
+          An array of values defining the configuration register for a given
+          pinmux mode. See fsl_iomuxc.h in the NXP HAL for a list.
+      nxp,software_input_on:
+        required: false
+        type: int
+        default: 0
+        enum:
+          - 0
+          - 1
+        description: |
+          00 Pin input follows peripheral
+          01 Pin is forced to input
+      nxp,hysteresis_enable:
+        required: false
+        type: int
+        default: 0
+        enum:
+          - 0
+          - 1
+        description: |
+          00 Hyst. Disable Field
+          01 Hyst. Enable Field
+      nxp,pull_up_down_config:
```

Comment:
please, use: `bias-pull-down`. List of options seems to only list a single pull-down resistor value.

Reply:
as in RT the two options is set to one set of bits, IMHO, keep the naming stupid and simple is better, rather than to fit into a enforced rules which just make the code hard to read.

Reply:
This topic was largely discussed and was done for a reason: standardize as much as possible (remember Zephyr is multiplatform) and to align with other projects (Linux) that already have gone through this problem. I'm afraid we can't now make exceptions because a vendor _doesn't like it_. Of course, you can always take the time to make an alternative proposal if you think the current solution needs to be improved.